netty/src/main/java/io/grpc/transport/netty/BufferingHttp2ConnectionEncoder.java

```diff
@@ -79,12 +79,19 @@
    */
   private final TreeMap<Integer, PendingStream> pendingStreams =
           new TreeMap<Integer, PendingStream>();
+  private final int initialMaxConcurrentStreams;
   // Smallest stream id whose corresponding frames do not get buffered.
   private int largestCreatedStreamId;
   private boolean receivedSettings;
 
   protected BufferingHttp2ConnectionEncoder(Http2ConnectionEncoder delegate) {
+    this(delegate, SMALLEST_MAX_CONCURRENT_STREAMS);
+  }
+
+  protected BufferingHttp2ConnectionEncoder(Http2ConnectionEncoder delegate,
+                                            int initialMaxConcurrentStreams) {
```

Comment:
Not at the moment.  I'm anticipating that this code will eventually become part of Netty and we may want the flexibility. Adding it now doesn't add too much complexity so I figured it wouldn't hurt.